### DISCUSSION TITLE
Default role coloring

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#458] [Feature Suggestions]
A common problem with discord is that people need "Member" roles as they want people do have a colored name,
This means they need to add a bot to the server to do so.
A cool feature would be to allow color changing on the Default role

Amazing project though!

**Comment by KartikeSingh** [comment]
That might be a nice feature, but did you heard about the feature in which people can like select options when they join the server and they can get a role.
So its like a select menu role built in with discord
